gluten-core/src/main/scala/io/glutenproject/utils/QueryPlanSelector.scala

```diff
@@ -43,12 +43,20 @@ abstract class QueryPlanSelector[T <: QueryPlan[_]] extends Logging {
     new Throwable().getStackTrace().slice(trim, trim + max).mkString("\n")
   }
 
+  private def isGlutenStatementEnable(session: SparkSession): Boolean = {
+    if (session.sparkContext.getLocalProperty(GlutenConfig.GLUTEN_ENABLE_FOR_THREAD_KEY) != null) {
```

Comment:
Nit:
Use a val to keep the value from `getLocalProperty`, and then re-use it in the below.